Fst2List.cpp

```diff
@@ -40,30 +42,31 @@ namespace unitex {
 const char
     * usage_Fst2List =
         "Usage:\n"
-          "Fst2List [-o outFile][-p s/f/d][-[a/t] s/m] [-m] [-d] [-f s/a][-s[0s] \"Str\"] [-r[s/l] \"Str\"] [-l line#] [-i subname]* [-c SS=0xxxx]* fname\n"
-          " fname : name of the input file name with extension \".fst2\"\r\n"
-          " -S : write path on standard output\r\n"
-          " -o outFile : if this option and -S not exist, save paths at \"file\"lst.txt\r\n"
-          " -[a/t] s/m : mode de automata or transducteur, s=single initial, m = multi-inital\r\n"
+          "Fst2List [-o outFile][-p s/f/d][-a/t s/m] [-m] [-d] [-f s/a] [--io-separator \"Str\"] [--stop-mark \"Str\"][-s \"Str\"] [-r s/l/x \"Str\"] [-l line#] [-i subname]* [-c SS=0xxxx]* fname\n"
+          " fname: input file name with extension \".fst2\"\r\n"
+          " -S: write path on standard output\r\n"
+          " -o outFile: if this option and -S are not used, save paths in \"file\"lst.txt\r\n"
+          " -a/t s/m, --automaton/transducer s/m: a = write outputs, t = ignore outputs, \r\n"
+          "     s=single initial state, m = multiples initals states\r\n"
           "              by default \"-a s\"\r\n"
-          " -l line#  :  max number of line to save[decimal].\r\n"
-          " -i sname : stop call of the exploitation at this sub-graphe \"sname\"\r\n"
-          " -p s/f/d : mode de extrait word, s=each sub graphs, f=full path(default),\r\n"
-          "            d= debugging. default is 'f'\r\n"
+          " -l line#: max number of line to save[decimal].\r\n"
+          " -i sname: stop exploration at subgraph \"sname\"\r\n"
+          " -p s/f/d: exploration mode, s=independently, printing names of called subgraphs, f=recursively(default),\r\n"
+          "            d=debugging. default is 'f'\r\n"
           " -c SS=0xXXXX: change the symbol string between symbols < and >,\"<SS>\" \r\n"
```

Comment:
`-c SS=0xXXXX: replaces symbol SS when it appears between angle brackets
by the Unicode character whose hexadecimal number is 0xXXXX;`